ui/test/callWithTelemetryAndErrorHandling.test.ts

```diff
@@ -63,4 +63,14 @@ suite('callWithTelemetryAndErrorHandling tests', () => {
             }),
             /testFuncErrorAsync/);
     });
+
+    test('mask', async () => {
```

Comment:
You should copy over all unit tests: https://github.com/microsoft/vscode-azuretools/blob/master/appservice/test/callWithMaskHandling.test.ts

Reply:
Oh man, I didn't expect me to actually have unit tests for this. Hooray past Nathan (and past Eric who probably made him make unit tests)